mlir/unittests/programs/qc_programs.cpp

```
@@ -1937,6 +1981,25 @@ SmallVector<Value> nestedForLoopWhileOp(QCProgramBuilder& b) {
   return measureAndReturn(b, reg.qubits);
 }
 
+SmallVector<Value> nestedForLoopSwitchOp(QCProgramBuilder& b) {
```

Cppcheck (2.21.0)

[style] The function 'nestedForLoopSwitchOp' is never used.

(unusedFunction)